### ISSUE TITLE
contrastive_top_1_accuracy and label_top_1_accuracyc，The training result is always 1

### ISSUE THREAD

**Issue by xiaoyi-st** [google-research/simclr#73]
Hi，Sorry to bother you again。When I was training, I found that these two results were always 1 and remained the same, so I could not make further progress in the training. Moreover, I did not know the difference between these two result contrastive_top_1_accuracy and label_top_1_accuracyc

**Comment by chentingpc** [comment]
contrastive acc is the accuracy of predicting positive augmentation pairs. And the label_top_1 is the accuracy of predicting the label. They are used differently during pretraining and finetuning. During the pretraining, you could ignore the label_top_1; and during fine-tuning / linear classifier training, you could ignore the contrastive accuracy.
